assets/javascripts/modules/validatorFocus.js

```diff
@@ -7,10 +7,26 @@
     }
 
     $('.error-summary a').on('click', function(e) {
+      var $this = $(this);
+      var focusId = $this.attr('data-focuses');
+      if (!focusId)
+      {
+        focusId = ("" + $this.attr("href")).trim();
```

Comment:
Yes, by default browser handles <a href="#some-id"> by jumping to the element. This module will now add a progressive enhanced behaviour to it by animating the transition.